srtcore/core.cpp

```diff
@@ -7183,6 +7182,8 @@ void CUDT::processCtrl(CPacket& ctrlpkt)
       }
       CGuard::leaveCS(m_AckLock);
 
+      updateCC(TEV_LOSSREPORT, EventVariant(losslist, losslist_len));
```

Comment:
@ethouris This relocation is done to have the `m_pSndLossList` with the newly reported losses. Those will be used to estimate the loss ratio as `pkts_lost / pkts_sent`